spi/src/main/java/org/eclipse/dataspaceconnector/spi/transfer/flow/DataReader.java

```diff
@@ -0,0 +1,9 @@
+package org.eclipse.dataspaceconnector.spi.transfer.flow;
+
+import org.eclipse.dataspaceconnector.spi.types.domain.transfer.DataAddress;
+
+public interface DataReader {
+    boolean canHandle(String type);
+
+    String read(DataAddress source);
```

Comment:
Maybe the return type should be enclosed in a Result that could report also errors? 
Maybe byte[] or InputStream would be better to handle a binary data type

Reply:
Done